### PR TITLE
Ks cancel fixes

### PR DESCRIPTION
**Pull request by captnseagraves**
Ks cancel fixes

### CODE REVIEW

**Comment by captnseagraves** [comment]
@mbeacom Just trying to fixes a few things in the cancel_subscription functionality, but have a bunch of other changes that come through from my migration issues last week. Do you have a suggested course of action to resolve the issues?

File: app/assets/v2/fonts/Muli/muli-v12-latin-600.svg [hunk 1]
```diff
@@ -1,309 +0,0 @@
-<?xml version="1.0" standalone="no"?>
```
Comment (mbeacom) [inline app/assets/v2/fonts/Muli/muli-v12-latin-600.svg]: Are these file deletions right for this branch?

Comment (captnseagraves) [reply app/assets/v2/fonts/Muli/muli-v12-latin-600.svg]: No I dont think so. not sure where they came through. should I just copy them from another branch and reimplement?

Comment (captnseagraves) [reply app/assets/v2/fonts/Muli/muli-v12-latin-600.svg]: @mbeacom file deletions undone, and named as in `Master`, my only question is on the migration files. Are those ok to add?

**Pull request merged**